src/dist/js/jquery-2.2.4.min.js

```diff
@@ -1,3 +1,3 @@
 /* NO, really, NOBODY NEEDS TO INCLUDE A FUCKING FRAMEWORK TO MAKE A COOL WEBSITE. */
-console.log('Don\'t worry, no jQuery was used in this website.');
-var $ = function(){this.fancyRainbow = function(){return this;};this.html = function(){return this;};return this;};
\ No newline at end of file
+console.log("Don't worry, no jQuery was used in this website.");
+var _=function(){return $},$=function(){$.fancyRainbow=_;$.html=_;return $}
```

Comment:
This change is going to [break this](https://github.com/denysvitali/thebestmotherfuckingwebsite/blob/master/src/pug/scriptjoke.html)  
I added this line to fake a jQuery function call

Reply:
Indeed, I was worried that there could be stack related issues, but turns out I was just being paranoid. Pushed a new commit to fix this, so there should only be one exported symbol to the global scope like before :wink: